src/maingoblint.ml

```diff
@@ -51,15 +51,19 @@ let rec option_spec_list: Arg_complete.speclist Lazy.t = lazy (
       set_string "outfile" "result";
     if get_string "exp.g2html_path" = "" then
       set_string "exp.g2html_path" (Fpath.to_string exe_dir);
-    set_bool "dbg.print_dead_code" true;
+    set_bool "ana.dead-code.lines" true;
+    set_bool "ana.dead-code.branches" true;
+    set_bool "ana.dead-code.functions" true;
     set_bool "exp.cfgdot" true;
     set_bool "g2html" true;
     set_string "result" "fast_xml"
   in
   let configure_sarif () =
     if (get_string "outfile" = "") then
       set_string "outfile" "goblint.sarif";
-    set_bool "dbg.print_dead_code" true;
+    set_bool "ana.dead-code.lines" true;
+    set_bool "ana.dead-code.branches" true;
+    set_bool "ana.dead-code.functions" true;
```

Comment:
Is this even required at all for SARIF? I guess it was just copied from the HTML case above.

Reply:
I removed it now.